bitnami/cert-manager/templates/webhook/service.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.metrics.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "certmanager.webhook.fullname" . }}
```

I'd this name since it's intended for metrics:
```suggestion
  name: {{ printf "%s-metrics" (include "certmanager.webhook.fullname" .) | trunc 63 | trimSuffix "-" }}
```

---

this service will be used for the cert-manager to verify the certificate, issuers, etc. We are taking the metrics just for the controller. I am going to skip this request.